masterlist.yaml

```diff
@@ -5432,7 +5435,7 @@ plugins:
         condition: 'active("Legendary Skyrim Crossbows.esp") and not active("Pan NPCs DG - Legendary Skyrim Crossbows.esp") and not active("Pan NPCs DG - Dawnguard Arsenal - Legendary Skyrim Crossbows.esp")'
       - <<: *patchProvided
         subs: [ 'AI Overhaul' ]
-        condition: 'active("AI Overhaul.esp") and not active("AI Overhaul - PAN_NPCs_DG Patch.esp") and not active("Bashed Patch.*\.esp")'
+        condition: 'active("AI Overhaul.esp") and not active("PAN_NPCs_DG - AI Overhaul Patch.esp") and not active("Bashed Patch.*\.esp")'
```

Comment:
Per https://github.com/loot/skyrimse/pull/2596#issuecomment-1287612914 (I have to do these as separate suggestions due to issues with GH):
```suggestion
```